src/test/regress/sql/distributed_functions.sql

```diff
@@ -449,6 +452,7 @@ TRUNCATE pg_dist_node;
 \c - - - :worker_2_port
 SET client_min_messages TO error; -- suppress cascading objects dropping
 UPDATE pg_dist_local_group SET groupid = 0;
+UPDATE pg_dist_node SET hasmetadata=false;
```

Comment:
Do we need this? Pg_dist_node gets truncated a few lines down and I'd expect it to not matter on the worker.

Reply:
I think we shouldn't need. This test is very complicated, we first do stop_metadata_sync on the coordinator, and trying to drop schema on the workers.

So, we could do `SET citus.enable_ddl_propagation TO off;` instead of  `UPDATE pg_dist_node SET hasmetadata=false;`, or I can try to clean-up the tests a bit.

I'll fix it tomorrow

Reply:
This hack is mostly because we're not dropping the schema on the workers when `DROP SCHEMA` is executed, and MX errors if you try to drop schema from the workers. I pushed a commit to improve a bit